static/app/views/starfish/views/webServiceView/starfishView.tsx

```diff
@@ -83,13 +83,14 @@ export function StarfishView(props: BasePerformanceViewProps) {
         start={eventView.start}
         end={eventView.end}
         organization={organization}
-        yAxis="equation|count_if(http.status_code,greaterOrEquals,500)/(count_if(http.status_code,equals,200)+count_if(http.status_code,greaterOrEquals,500))"
+        yAxis="http_error_count()"
+        dataset={DiscoverDatasets.METRICS}
       >
         {eventData => {
           const transformedData: Series[] | undefined = eventData.timeseriesData?.map(
             series => ({
               data: series.data,
-              seriesName: t('Error Rate'),
+              seriesName: t('Errors (5XXs)'),
```

Comment:
I added 5XX rate to the api module in this PR  https://github.com/getsentry/sentry/pull/50243
Is there a specific reason we would use `count` here instead of rate? I think this should be consistent between the two.

Reply:
I switched to count because that's what Dora had in her mocks 🤔

Reply:
Good point, lemme try to figure out which one we should use.

Reply:
Just to circle back the conversation from slack to here, we're going with counts

Reply:
Sounds good! i've updated my PR to use count.